go.mod

```diff
@@ -1,104 +1,110 @@
 module github.com/buildbarn/bb-remote-asset
 
-go 1.21
+go 1.23.0
+
+toolchain go1.23.1
 
 // Using the most recent version causes a cyclic dependency in protoc.
 replace google.golang.org/protobuf => google.golang.org/protobuf v1.32.0
 
 require (
-	github.com/bazelbuild/buildtools v0.0.0-20240313121412-66c605173954
-	github.com/bazelbuild/remote-apis v0.0.0-20240319211552-96942a2107c7
-	github.com/buildbarn/bb-storage v0.0.0-20240331131648-914e53aad8cd
-	github.com/golang/mock v1.6.0
-	github.com/prometheus/client_golang v1.19.0
+	github.com/bazelbuild/buildtools v0.0.0-20240918101019-be1c24cc9a44
+	github.com/bazelbuild/remote-apis v0.0.0-20240926071355-6777112ef7de
+	github.com/buildbarn/bb-storage v0.0.0-20240928104605-8abbcfab01bc
+	github.com/golang/mock v1.7.0-rc.1
```

Comment:
We should probably migrate to go.uber.org/mock like in bb-storage [here](https://github.com/buildbarn/bb-storage/commit/24d544b34a866f55838d576a0b110f4495f04450)

Reply:
@mortenmj what do you think? ^

Reply:
(no one asked, but) I think it'd be OK to do this in a separate change - what do you all think?